### PR TITLE
2.0.6 parity

### PR DESCRIPTION
**Pull request by joshsloat**
Introduces changes from #80 and #82 into v3.

### CODE REVIEW

**Comment by joshsloat** [comment]
Note to selves: Close out #79 and #81 once 3.0.2 is released.

File: stories/ShouldHandleZoom.js [hunk 1]
```diff
@@ -28,8 +28,9 @@ const ShouldHandleZoom = ({ isEnabled, onToggle }) =>
       shouldHandleZoom prop which should be a function that returns a boolean.
     </p>
     <p>
-      Try it! Click this checkbox to enable zooming and uncheck it to disable
-      the zoom capabilities.
+      Try it! Click this checkbox to enable zooming{' '}
+      <strong>only when the image is clicked </strong>
```
Comment (rpearce) [inline stories/ShouldHandleZoom.js]: strange requirement for a space on the line above and before the closing strong tag here...

**Comment by rpearce** [comment]
lgtm! going to merge, test a bit more, add some changelog stuff, and ship it

File: stories/ShouldHandleZoom.js (hunk 1 above)
Comment (joshsloat) [reply stories/ShouldHandleZoom.js]: This text doesn't break in the browser based on the line breaks here. This was needed to provide space between "clicked" and "and".

**Pull request merged**